lib/ansible/modules/cloud/amazon/ec2_vpc_peer.py

```diff
@@ -356,6 +384,7 @@ def main():
         dict(
             vpc_id=dict(),
             peer_vpc_id=dict(),
+            peer_region=dict(),
```

Comment:
I think if someone is using this option but has an older version of botocore a helpful error should be added.